hnn_core/drives.py

```diff
@@ -197,7 +197,7 @@ def _add_drives_from_params(net):
         drive['event_seed'] += net.gid_ranges[drive_name][0]
 
 
-def _get_prng(seed, gid, sync_evinput=False):
```

Comment:
don't we need `sync_evinput` ? I'm a bit confused by this change. See: https://github.com/jonescompneurolab/hnn-core/blob/00ebb0fff014c32d0a6b151c45a5ed36cce7c284/hnn_core/param/gamma_L5weak_L2weak.json#L198

Reply:
Nope, this functionality is now handled by setting `n_drive_cells=1` and `cell_specific=False` in [`Network.add_evoked_drive`](https://github.com/jonescompneurolab/hnn-core/blob/2e5382781ba8ed780e1120556a27184924635a7e/hnn_core/network.py#L499). Instead of assigning the same rng state to multiple artificial drive cells to create a "synchronous drive", we now create one artificial drive cell with a single rng state that is connected to all cells.

Note also that this change is reflected in [`_extract_drive_specs_from_hnn_params()`](https://github.com/jonescompneurolab/hnn-core/blob/2e5382781ba8ed780e1120556a27184924635a7e/hnn_core/params.py#L207), and if you grep `_get_prng`, you'll find it's never used with `sync_evinput=False`.